design-document.md

```diff
@@ -477,12 +477,17 @@ uploading, open issues, and system parameters?]]
 
 ### The input-validation protocol
 
-**Notation.**
-Fix a finite field K. We let K^n denote the set of vectors over K of length n.
-We write the linear, s-way secret sharing of an element x of K^n as {x:1}, ...,
-{x:s}, where {x:i} is the share held by the i-th party. We write {x} as
-shorthand for the sequence {x:1}, ..., {x:s}. Each {x:i} is also an element of
-K^n.
+As described in {{overview}} each client wishes to report a set of
+inputs. For simple measurements, such as having each client report a
+count and computing the total, each input is just mapped onto a single
+integer. For more complicated measurements, it may be necessary to map
+a given input onto multiple integers. In any case, the client takes
+all of its inputs and maps them onto a vector of integers each 
+of which is a member of a finite field K. We denote this vector as x.
```

Comment:
I understand the intention of this paragraph, however I think it's a bit confusing as it is. For instance, you say that a client wishes to report a "set of inputs", whereas above we think of each clients sending exactly one input, and we refer to the inputs of the aggregation function collectively as the "set of inputs"

WDYTA:
> The Prio computation is parameterized by a finite field, which we will call K, and an integer n. Each client encodes its input as a length-n vector of element of K. The length of the vector depends on the type of data being collected. A single field element may be sufficient for some applications, whereas more sophisticated measurements will require larger encodings.

Reply:
Perhaps this is just me being confused. When you and I talked the other day, what I thought I heard was that if I wanted to report, say, "number of urls" and "number of clicks", I would turn that into a single 2-length vector of integers rather then sending two entirely different 1-length vectors and proofs.' Is that not correct?

Reply:
That's correct. My point is that, in the language of this doc, the "input" in this case is the pair (number_of_urls, number_of_clicks). I'm trying to be precise about the language.

Reply:
Taking my suggestion.